testool/src/statetest/report.handlebars

```diff
@@ -1,6 +1,9 @@
 <HTML>
 <HEAD>
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton-framework/1.1.1/skeleton.css">
+<style>
+table#table tr td:first-child { display: none; }
```

Comment:
Thanks for catch up this, just fix the missing header.
This `id` column is for supporting hashtag future feature, sth like `https://jenkins.zkevm-testnet.org/job/testool/Testool_20Report/#<hash tag>`.
But since this feature not support yet, I will hide first column first